tests/components/hue/conftest.py

```diff
@@ -41,10 +42,25 @@ def create_mock_bridge(hass):
     bridge.mock_group_responses = bridge.api.mock_group_responses
     bridge.mock_sensor_responses = bridge.api.mock_sensor_responses
 
+    async def async_setup():
```

Comment:
Mocking our bridge implementation instead of just the library bridge creates maintenance load and risk. I'd advice, for the future, to rewrite the tests to not interact with integration details.